core/src/types/operator/blocking_operator.rs

```diff
@@ -168,6 +170,12 @@ impl BlockingOperator {
     ///
     /// Refer to [RFC: List Prefix][crate::docs::rfcs::rfc_3243_list_prefix] for more details.
     ///
+    /// ## Services that not support `create_dir`
+    ///
+    /// For services that not support `create_dir`, `stat("test/")` will return `NotFound` even
+    /// when `test/abc` exists since the service won't have the concept of dir. There is nothing
+    /// we can do about this.
```

Comment:
This should also be added to the upgrade guide.

Reply:
> This should also be added to the upgrade guide.

Why? Behavior not changed.

Reply:
For now, there is no content about whether these services that don't support create_dir are affected or not in the guide.

Reply:
Welcome to add them, but I don't think they are breaking changes.